src/Chainweb/Pact/PactService/Checkpointer.hs

```diff
@@ -645,11 +645,12 @@ failOnTooLowRequestedHeight
     -> Maybe RewindLimit
     -> BlockHeader
     -> PactServiceM logger tbl ()
-failOnTooLowRequestedHeight parent (Just limit) lastHeader
-    | parentHeight + 1 + limitHeight < lastHeight = -- need to stick with addition because Word64
-        throwM $ RewindLimitExceeded limit parentHeight lastHeight parent
-  where
-    limitHeight = BlockHeight $ _rewindLimit limit
-    parentHeight = _blockHeight parent
-    lastHeight = _blockHeight lastHeader
-failOnTooLowRequestedHeight _ _ _ = return ()
+failOnTooLowRequestedHeight parent mLimit lastHeader = case mLimit of
```

Comment:
I had refactored this function when adding the new exception check, but it wasn't necessary.